src/common/textcmn.cpp

```diff
@@ -163,6 +163,8 @@ void wxTextAttr::Init()
     m_fontStyle = wxFONTSTYLE_NORMAL;
     m_fontWeight = wxFONTWEIGHT_NORMAL;
     m_fontUnderlined = false;
+    m_fontUnderlinedType = wxTEXT_ATTR_UNDERLINE_NONE;
+    m_fontUnderlineColor = *wxWHITE;
```

Comment:
Why? This shouldn't be necessary and white is almost surely not the right colour to use.

Reply:
OK